app/usecases/anticheat.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import inspect
+import sys
+from typing import Optional
+
+import app.logging
+import app.state.sessions
+from app.constants.gamemodes import GameMode
+from app.constants.mods import Mods
+from app.constants.privileges import Privileges
+from app.objects.player import Player
+from app.objects.score import Score
+
+# DO NOT import functions directly, it'll mess up inspect.getmembers
+
+# Add checks here that you'd like to have disabled, but not removed from the code entirely
+disabled_checks = []
+
+
+async def run_anticheat_checks(player: Player, score: Score):
+    checks = dict(inspect.getmembers(sys.modules[__name__], inspect.isfunction))
```

Comment:
core idea is good (i've had similar thought before) but implementation is a bit weird here

better to make this either an (optional) separate service entirely, or at least a module that's under gitignore